<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1013</width>
    <height>701</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Grouping_Base</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <property name="leftMargin">
     <number>5</number>
    </property>
    <property name="topMargin">
     <number>5</number>
    </property>
    <property name="rightMargin">
     <number>5</number>
    </property>
    <property name="bottomMargin">
     <number>5</number>
    </property>
    <property name="spacing">
     <number>5</number>
    </property>
    <item row="0" column="0">
     <layout class="QGridLayout" name="pGridLayout_Common">
      <property name="spacing">
       <number>5</number>
      </property>
      <item row="0" column="0">
       <widget class="QTabWidget" name="pTabWidget">
        <property name="currentIndex">
         <number>0</number>
        </property>
        <widget class="QWidget" name="pTab_Imperx">
         <attribute name="title">
          <string>Imperx</string>
         </attribute>
         <layout class="QGridLayout" name="gridLayout_3">
          <property name="leftMargin">
           <number>5</number>
          </property>
          <property name="topMargin">
           <number>5</number>
          </property>
          <property name="rightMargin">
           <number>5</number>
          </property>
          <property name="bottomMargin">
           <number>5</number>
          </property>
          <property name="spacing">
           <number>5</number>
          </property>
          <item row="1" column="0">
           <layout class="QGridLayout" name="pGridLayout_Imperx_"/>
          </item>
          <item row="0" column="0">
           <widget class="QFrame" name="pFrame_Imperx">
            <property name="frameShape">
             <enum>QFrame::StyledPanel</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Raised</enum>
            </property>
            <layout class="QGridLayout" name="pGridLayout_Imperx_2">
             <property name="leftMargin">
              <number>5</number>
             </property>
             <property name="topMargin">
              <number>5</number>
             </property>
             <property name="rightMargin">
              <number>5</number>
             </property>
             <property name="bottomMargin">
              <number>5</number>
             </property>
             <property name="spacing">
              <number>5</number>
             </property>
             <item row="0" column="0">
              <layout class="QGridLayout" name="pGridLayout_Imperx">
               <property name="spacing">
                <number>5</number>
               </property>
               <item row="0" column="0">
                <widget class="QFrame" name="pFrame_FrameChoice">
                 <property name="frameShape">
                  <enum>QFrame::StyledPanel</enum>
                 </property>
                 <property name="frameShadow">
                  <enum>QFrame::Plain</enum>
                 </property>
                 <layout class="QGridLayout" name="gridLayout_9">
                  <property name="leftMargin">
                   <number>5</number>
                  </property>
                  <property name="topMargin">
                   <number>5</number>
                  </property>
                  <property name="rightMargin">
                   <number>5</number>
                  </property>
                  <property name="bottomMargin">
                   <number>5</number>
                  </property>
                  <property name="spacing">
                   <number>5</number>
                  </property>
                  <item row="0" column="0">
                   <layout class="QGridLayout" name="pGridLayout_FrameChoice">
                    <property name="spacing">
                     <number>5</number>
                    </property>
                   </layout>
                  </item>
                 </layout>
                </widget>
               </item>
               <item row="0" column="1">
                <widget class="QFrame" name="pFrame_Timing">
                 <property name="frameShape">
                  <enum>QFrame::StyledPanel</enum>
                 </property>
                 <property name="frameShadow">
                  <enum>QFrame::Plain</enum>
                 </property>
                 <layout class="QGridLayout" name="gridLayout_11">
                  <property name="leftMargin">
                   <number>5</number>
                  </property>
                  <property name="topMargin">
                   <number>5</number>
                  </property>
                  <property name="rightMargin">
                   <number>5</number>
                  </property>
                  <property name="bottomMargin">
                   <number>5</number>
                  </property>
                  <property name="spacing">
                   <number>5</number>
                  </property>
                 </layout>
                </widget>
               </item>
               <item row="1" column="0">
                <widget class="QFrame" name="pFrame_Commands">
                 <property name="frameShape">
                  <enum>QFrame::StyledPanel</enum>
                 </property>
                 <property name="frameShadow">
                  <enum>QFrame::Plain</enum>
                 </property>
                 <layout class="QGridLayout" name="gridLayout_18">
                  <property name="leftMargin">
                   <number>5</number>
                  </property>
                  <property name="topMargin">
                   <number>5</number>
                  </property>
                  <property name="rightMargin">
                   <number>5</number>
                  </property>
                  <property name="bottomMargin">
                   <number>5</number>
                  </property>
                  <property name="spacing">
                   <number>5</number>
                  </property>
                  <item row="0" column="0">
                   <layout class="QGridLayout" name="pGridLayout_Commands">
                    <property name="spacing">
                     <number>5</number>
                    </property>
                    <item row="1" column="0">
                     <widget class="QLabel" name="pLabel_Address">
                      <property name="font">
                       <font>
                        <family>Arial</family>
                        <pointsize>10</pointsize>
                       </font>
                      </property>
                      <property name="text">
                       <string>Address</string>
                      </property>
                      <property name="alignment">
                       <set>Qt::AlignCenter</set>
                      </property>
                     </widget>
                    </item>
                    <item row="1" column="1">
                     <widget class="QLabel" name="pLabel_Command">
                      <property name="font">
                       <font>
                        <family>Arial</family>
                        <pointsize>10</pointsize>
                       </font>
                      </property>
                      <property name="text">
                       <string>Command</string>
                      </property>
                      <property name="alignment">
                       <set>Qt::AlignCenter</set>
                      </property>
                     </widget>
                    </item>
                    <item row="2" column="0">
                     <widget class="QLineEdit" name="pLineEdit_Address">
                      <property name="font">
                       <font>
                        <family>Arial</family>
                        <pointsize>10</pointsize>
                       </font>
                      </property>
                      <property name="inputMask">
                       <string>\0\x&gt;HHHH</string>
                      </property>
                      <property name="text">
                       <string>0x0100</string>
                      </property>
                      <property name="alignment">
                       <set>Qt::AlignCenter</set>
                      </property>
                     </widget>
                    </item>
                    <item row="2" column="1">
                     <widget class="QLineEdit" name="pLineEdit_Command">
                      <property name="font">
                       <font>
                        <family>Arial</family>
                        <pointsize>10</pointsize>
                       </font>
                      </property>
                      <property name="inputMask">
                       <string>\0\x&gt;HHHHHHHH</string>
                      </property>
                      <property name="text">
                       <string>0x00000001</string>
                      </property>
                      <property name="alignment">
                       <set>Qt::AlignCenter</set>
                      </property>
                     </widget>
                    </item>
                    <item row="3" column="0" colspan="2">
                     <widget class="QPushButton" name="pPushButton_SendCommand">
                      <property name="font">
                       <font>
                        <family>Arial</family>
                        <pointsize>10</pointsize>
                       </font>
                      </property>
                      <property name="text">
                       <string>Send Command</string>
                      </property>
                     </widget>
                    </item>
                    <item row="0" column="0" colspan="2">
                     <widget class="QLabel" name="pLabel_Commands_Title">
                      <property name="font">
                       <font>
                        <family>Arial</family>
                        <pointsize>10</pointsize>
                        <weight>75</weight>
                        <bold>true</bold>
                       </font>
                      </property>
                      <property name="text">
                       <string>Комманды (Commands)</string>
                      </property>
                      <property name="alignment">
                       <set>Qt::AlignCenter</set>
                      </property>
                     </widget>
                    </item>
                   </layout>
                  </item>
                 </layout>
                </widget>
               </item>
               <item row="1" column="1">
                <widget class="QFrame" name="pFrame_Info">
                 <property name="frameShape">
                  <enum>QFrame::StyledPanel</enum>
                 </property>
                 <property name="frameShadow">
                  <enum>QFrame::Plain</enum>
                 </property>
                 <layout class="QGridLayout" name="gridLayout_19">
                  <property name="leftMargin">
                   <number>5</number>
                  </property>
                  <property name="topMargin">
                   <number>5</number>
                  </property>
                  <property name="rightMargin">
                   <number>5</number>
                  </property>
                  <property name="bottomMargin">
                   <number>5</number>
                  </property>
                  <property name="spacing">
                   <number>5</number>
                  </property>
                  <item row="0" column="0">
                   <layout class="QGridLayout" name="pGridLayout_Info">
                    <property name="spacing">
                     <number>5</number>
                    </property>
                    <item row="0" column="0">
                     <widget class="QLabel" name="pLabel_FPS_Title">
                      <property name="font">
                       <font>
                        <family>Arial</family>
                        <pointsize>10</pointsize>
                        <weight>75</weight>
                        <bold>true</bold>
                       </font>
                      </property>
                      <property name="text">
                       <string>Частота кадров, FPS</string>
                      </property>
                     </widget>
                    </item>
                    <item row="1" column="0">
                     <widget class="QLabel" name="pLabel_TimerCounter">
                      <property name="font">
                       <font>
                        <family>Arial</family>
                        <pointsize>10</pointsize>
                       </font>
                      </property>
                      <property name="text">
                       <string>-1</string>
                      </property>
                     </widget>
                    </item>
                    <item row="0" column="1">
                     <widget class="QLabel" name="pLabel_FPS">
                      <property name="font">
                       <font>
                        <family>Arial</family>
                        <pointsize>10</pointsize>
                        <weight>75</weight>
                        <bold>true</bold>
                       </font>
                      </property>
                      <property name="text">
                       <string>0</string>
                      </property>
                     </widget>
                    </item>
                    <item row="1" column="1">
                     <widget class="QLabel" name="pLabel_FrameCounter">
                      <property name="font">
                       <font>
                        <family>Arial</family>
                        <pointsize>10</pointsize>
                       </font>
                      </property>
                      <property name="text">
                       <string>-1</string>
                      </property>
                     </widget>
                    </item>
                    <item row="2" column="0" colspan="2">
                     <widget class="QCheckBox" name="pCheckBox_FrameRecording">
                      <property name="font">
                       <font>
                        <family>Arial</family>
                        <pointsize>10</pointsize>
                       </font>
                      </property>
                      <property name="text">
                       <string>Запись кадров (Frame Recording)</string>
                      </property>
                     </widget>
                    </item>
                   </layout>
                  </item>
                 </layout>
                </widget>
               </item>
              </layout>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
     </layout>
    </item>
    <item row="1" column="0">
     <widget class="QFrame" name="pFrame_EthernetAdapter">
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Plain</enum>
      </property>
      <layout class="QGridLayout" name="gridLayout_8">
       <property name="leftMargin">
        <number>5</number>
       </property>
       <property name="topMargin">
        <number>5</number>
       </property>
       <property name="rightMargin">
        <number>5</number>
       </property>
       <property name="bottomMargin">
        <number>5</number>
       </property>
       <property name="spacing">
        <number>5</number>
       </property>
       <item row="0" column="0">
        <layout class="QGridLayout" name="pGridLayout_EthernetAdapter">
         <property name="spacing">
          <number>5</number>
         </property>
         <item row="1" column="1">
          <widget class="QPushButton" name="pPushButton_DeviceInitialization">
           <property name="font">
            <font>
             <family>Arial</family>
             <pointsize>10</pointsize>
            </font>
           </property>
           <property name="text">
            <string>Инициализация адаптера (Device Initialization)</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QComboBox" name="pComboBox_DeviceChoice"/>
         </item>
         <item row="2" column="0">
          <widget class="QPushButton" name="pPushButton_Start">
           <property name="font">
            <font>
             <family>Arial</family>
             <pointsize>10</pointsize>
            </font>
           </property>
           <property name="text">
            <string>Начало обмена (Start)</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QPushButton" name="pPushButton_Stop">
           <property name="font">
            <font>
             <family>Arial</family>
             <pointsize>10</pointsize>
            </font>
           </property>
           <property name="text">
            <string>Конец обмена (Stop)</string>
           </property>
          </widget>
         </item>
         <item row="0" column="0" colspan="2">
          <widget class="QLabel" name="pLabel_DeviceChoice">
           <property name="font">
            <font>
             <family>Arial</family>
             <pointsize>12</pointsize>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>Выбор адаптера (Device Choice)</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1013</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
